src/index.js

```diff
@@ -21,7 +21,7 @@ app.use(cors());
 // Normal express config defaults
 app.use(morgan('dev'));
 
-app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
Is there a reason we are setting this to `true`?

Reply:
yes...so that the req.body can be either in json format or a formdata when testing from postman